### DISCUSSION TITLE
Anagrafica - Registrazione

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#336] [Idee e proposte]
Salve, sarebbe bello se fossero i genitori, attraverso un apposito modulo, a registrare i propri figli sul registro così da far risparmiare tempo alla segreteria... Per quanto riguarda l'approvazione dell'iscrizione sarebbe bello lasciare alla segreteria un tasto per approvare o rifiutare l'iscrizione dei genitori/alunni sul registro.

**Comment by trinko** [comment]
La segno come richiesta
